scifin/classifier/classifier.py

```diff
@@ -197,42 +212,79 @@ def kmeans_base_clustering(corr, names_features=None, max_num_clusters=10, n_ini
 
 
 
-
-def make_new_outputs(corr, clusters, clusters2):
+@typechecked
+def make_new_outputs(corr: Union[np.array, pd.DataFrame], clusters: dict, clusters2: dict)\
+        -> (pd.DataFrame, dict, pd.Series):
```

Comment:
same here

Reply:
I've found a format that I like a little better, but we can correct that easily some day.